docs/reference/administration_reference.md

```diff
@@ -0,0 +1,82 @@
+| **display_delete**         |            | *True*                                 | toggles Delete button in details view                                                         |
+| **item_field_list**        | *required* |                                        | list of items to be rendered in the details page                                              |
+
+## Create View
+
+### Attributes
+
+| Attribute           | Required   | Default                              | Description                                                                                      |
+|---------------------|------------|--------------------------------------|--------------------------------------------------------------------------------------------------|
+| **name**            |            | *view class name*                    | view's name used by Flask to create routing                                                      |
+| **url**             |            | *view's 'name'*                      | url used to route the view.                                                                      |
+| **resource_config** | *required* |                                      | used to retrieve the resource (refers to extension attribute - check ext.py)                     |
+| **pid_path**        |            | *pid*                                | used for accessing the unique identifier of a resource                                           | 
+| **api_endpoint**    | *required* |                                      | used to execute POST request.                                                                    |
+| **title**           | *required* |                                      | displays a title in the view page.                                                               |
+| **list_view_name**  | *required* |                                      | name of the resource's list view name, enables  navigation between create modal and list view.   |
+| **form_fields**     | *required* |                                      | fields to be displayed  in the creation form.                                                    |                                                
+| **template**        |            | "invenio_administration/create.html" | template to be rendered for the view. Can be overridden by a custom template.                    |
+
+## Edit View
+
+### Attributes
+
+| Attribute           | Required   | Default                               | Description                                                                                 |
+|---------------------|------------|---------------------------------------|---------------------------------------------------------------------------------------------|
+| **name**            |            | *view class name*                     | view’s name used internally by Flask                                                        |
```

Comment:
Name is required.

Reply:
if the name is not given, then it is generated automatically, that's why it is not required. Did something change?